resources/dicts/thoughts/alive/general.json

```
@@ -521,7 +535,10 @@
         "id": "ambitious_to_leader",
         "thoughts": [
             "Is asking the Clan leader what {PRONOUN/m_c/subject} can do to help out around camp",
-            "Has been imitating the Clan leader's behavior recently"
+            "Has been imitating the Clan leader's behaviour recently",
+            "Knows {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} a shoo-in for deputy",
+            "Wonders if the Clan leader talks about {PRONOUN/m_c/object} when {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} not around",
+            "Tells the Clan leader {PRONOUN/lead_name/subject}{VERB/lead_name/'ve/'s} been doing a wonderful job leading"
```

```suggestion
            "Tells lead_name that {PRONOUN/lead_name/subject}{VERB/lead_name/'ve/'s} been doing a wonderful job leading"
```
for readability, having 'leader' and 'leading' in the same sentence just feels wrong imo